FirebaseSessions/Sources/ApplicationInfo.swift

```diff
@@ -63,4 +75,12 @@ class ApplicationInfo: ApplicationInfoProtocol {
   var mccMNC: String {
     return FIRSESValidateMccMnc(networkInfo.mobileCountryCode, networkInfo.mobileNetworkCode) ?? ""
   }
+  
+  var environment: DevEnvironment {
+    let environment = envParams["FIREBASE_RUN_ENVIRONMENT"]
```

Comment:
Maybe make this a Sessions specific name? Eg. `-FIRSessionsRunEnvironment` or `FIREBASE_SESSIONS_RUN_ENVIRONMENT` if you want to keep it a plist.

Could we also make this a constant, eg:   `private let commandLineArgument = "-FIRSessionsRunEnvironment"` in this class

Reply:
I like this idea. Will go ahead with "FIRSessionsRunEnvironment" as the name of the environment variable.